build/source/driver/multi_driver.f90

```diff
@@ -32,8 +32,9 @@ program multi_driver
 USE ascii_util_module,only:file_open                        ! open ascii file
 USE ascii_util_module,only:get_vlines                       ! read a vector of non-comment lines from an ASCII file
 USE ascii_util_module,only:split_line                       ! extract the list of variable names from the character string
-USE allocspace_module,only:initStruct                       ! module to allocate space for data structures
-USE allocspace_module,only:allocLocal                       ! module to allocate space for data structures
+USE allocspace_module,only:allocate_gru_struc               ! read/allocate space for gru-hru mask structures
+USE allocspace_module,only:allocGlobal                      ! module to allocate space for global data structures
+USE allocspace_module,only:allocLocal                       ! module to allocate space for local data structures
```

Comment:
Yes, this will be useful, though it should not hold up the merging of this PR. I've opened an issue so this can be addressed.